pkgs/applications/misc/blender/default.nix

```diff
@@ -30,15 +30,19 @@ let
     url = "https://developer.download.nvidia.com/redist/optix/v7.3/OptiX-7.3.0-Include.zip";
     sha256 = "0max1j4822mchj0xpz9lqzh91zkmvsn4py0r174cvqfz8z8ykjk8";
   };
+  libdecor' = libdecor.overrideAttrs (old: {
+    # Blender uses private APIs, need to patch to expose them
+    patches = (old.patches or [ ]) ++ [ ./libdecor.patch ];
+  });
```

Comment:
I think, in C you are allowed to call implicitly-defined functions as long as they return int. Perhaps that's what they are relying on and we are using some strict flags? It would be nice to have that reported upstream along with the compilation error.